website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4679,6 +4695,13 @@ trigger. To set a window to discard fired panes, set `accumulation_mode` to
 `DISCARDING`.
 {{< /paragraph >}}
 
+{{< paragraph class="language-go" >}}
+To set a window to accumulate the panes that are produced when the trigger
+fires, set the `AccumulationMode{Mode:}` parameter to `AccumulationMode_ACCUMULATING` when you set the
```

Comment:
The phrasing here is awkward. Don't try to present it as setting a parameter in these instances, demonstrate the code directly instead. 

```
To set a window to accumulate the panes that are produced when the trigger
fires, use `beam.AccumulationMode{Mode: window.Accumulating}`.
```
etc.

Reply:
It's not 100% clear to me whether we can make this a bit shorter for users by having convenience methods, other than using separate convenience method names: `beam.PanesAccumulate`, `beam.PanesDiscard`,   (ignoring `beam.PanesRetract` for now since nothing supports it IIRC...)